github.com.conf.template

```diff
@@ -36,18 +28,20 @@ server {
     # for the Digital Ocean resolver
     resolver ${RESOLVER};
 
-    # only opensafely, opensafely-core, and opensafely-actions orgs can be accessed
-    # location regex cannot match on query parameters
+    # only the allowed orgs (currently opensafely, opensafely-core, and
+    # opensafely-actions) can be accessed
 
     # basic git metadata operations
+    # location regex cannot match on query parameters
     location ~ ^/(${ORGS})/[^/]+/info/refs {
         limit_except GET { deny all; }
         if ($args !~ "service=git-upload-pack") { return 403; }
         proxy_pass https://github.com;
         proxy_redirect default;
     }
 
-    # git clone
+    # git clone requires a POST, sadly so we allow it to the specific url
+    # we are protected somewhat by the client_max_body_size limit
```

Comment:
Protected from what?

Reply:
from using it as way to ship data out via the proxy via POST